README.md

```diff
@@ -1,70 +1,78 @@
 # WordPress Plugin Boilerplate Powered
+
 <p align="center">
-<img src="https://raw.githubusercontent.com/WPBP/boilerplate-assets/master/icon-256x256.png" alt="Logo" title="Logo">
+  <img src="https://raw.githubusercontent.com/WPBP/boilerplate-assets/master/icon-256x256.png" alt="Logo">
 </p>
 
 [![Backers on Open Collective](https://opencollective.com/WordPress-Plugin-Boilerplate-Powered/backers/badge.svg)](#backers)
 [![Sponsors on Open Collective](https://opencollective.com/WordPress-Plugin-Boilerplate-Powered/sponsors/badge.svg)](#sponsors)
 
 **WordPress Plugin Boilerplate Powered** is a complete foundation for building your WordPress plugins following PSR-4 standard.  
-The project history is part of the [Wiki](https://wpbp.github.io/wiki.html) that include also the workflow and how it is organized.  
+The project history is part of the [Wiki](https://wpbp.github.io/wiki.html) that also includes the workflow and how it is organized.
 
-Every experienced WordPress developers use always the same libraries (or snippets) but at same time follow best practices.  
-Choose a library or a snippet and the boilerplate (with the help of the [Code Generator](https://wpbp.github.io/wiki.html)) will generate everything without need a manual copy&paste way or look on internet!
+Every experienced WordPress developer uses always the same libraries (or snippets) but at the same time follows best practices.  
+Choose a library or a snippet and the boilerplate (with the help of the [Code Generator](https://wpbp.github.io/wiki.html))
+will generate everything without the need of manual copy&paste way or searching on the Internet!
 
 ## Features
 
-### Libraries and Miscellaneous available
+### Available Libraries and other stuff
```

Comment:
Or "Included"??

Reply:
Well I prefer available at this point, because with the generator is possible to remove the ones that are not needed.